src/Command/Fry.js

```diff
@@ -10,12 +10,16 @@ const messages = {
 module.exports = Command.extend({
     commandName: 'fry',
     advertisable: false,
+    config: null,
+    dependencies: {
+        'config': 'config',
```

Comment:
This isn't necessary as dependencies are merged with their parent class.  Please remove it.